resources/js/Components/Navigation.svelte

```diff
@@ -1,19 +1,163 @@
 <script>
     import { inertia } from "@inertiajs/svelte";
     import Logo from "@components/Logo.svelte";
+    const links = {
+        discord: "https://discord.gg/tCg94qy",
+        wiki: "https://wiki.abidanarchive.com",
+    };
```

Comment:
I don't mind us putting these here, especially if they're just in the nav.
If we put the same links in the footer, then we should make a json file or a links service file that has a single source of truth.

Reply:
We shouldn't need them in the footer if we have a sticky nav, I think